src/Debian/Debianize/SourceDebDescription.hs

```diff
@@ -99,7 +99,7 @@ newSourceDebDescription =
       , _homepage = Nothing
       , _vcsFields = Set.empty
       , _xFields = Set.empty
-      , _xDescription = Nothing -- Quick hack, I should maybe put this into _xFields
+      , _description = Nothing -- Quick hack, I should maybe put this into _xFields
```

Comment:
David's comment could probably be deleted too

Reply:
No, I think the only problem left to resolve is that you've introduced a naming clash in the re-exports of Debian.Debianize

Reply:
Would it be acceptable to qualify the `imports` with `S` and `B`, as they are in `Tests.hs`?